code/game/objects/structures/statues.dm

```diff
@@ -213,15 +225,18 @@
 	desc = "This is a very expensive diamond statue."
 
 /obj/structure/statue/diamond/captain
-	name = "statue of THE captain"
+	name = "statue of the captain"
+	desc = "This is a very expensive diamond statue of The Captain."
```

Comment:
Why the capitals?

Reply:
I thought it looked better stylistically. Originally it said "THE captain".